app.js

```diff
@@ -31,10 +31,10 @@ Additionnal commands include:
  - \`/restart\`: Recreate the discord bot client.
  - \`/delete n\`: Delete the last n messages. Use without n to delete only the last one.
  - \`/set target value\`: Set a parameter (\`target\`) to \`value\`. Available parameters are:
-    - \`minmsissue_number (default value: \`100\`): The minimum issue number required in #number before linking to an issue on musescore.org.
+    - \`minmsissue_number\` (default value: \`100\`): The minimum issue number required in #number before linking to an issue on musescore.org.
     - \`minghissue_number (default value: \`100\`): The minimum issue number required in #number before linking to an issue on github.
```

Comment:
Same for minghissue_number maybe?

Reply:
Or actually, I'll do it.

Reply:
Yes, there too, not sure why I missed that